contracts/src/everlong/EverlongERC4626.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+// import { IEverlongERC4626 } from "../interfaces/IEverlongERC4626.sol";
+import { ERC4626 } from "solady/tokens/ERC4626.sol";
+
+contract EverlongERC4626 is ERC4626 {
+    /// @notice Whether virtual shares will be used to mitigate the inflation attack.
+    bool public constant useVirtualShares = true;
```

Comment:
I think this comment could be more concrete since we ARE using virtual shares.

```suggestion
    /// @notice Everlong uses virtual shares to mitigate inflation attacks
    bool public constant useVirtualShares = true;
```